bic

```diff
@@ -301,9 +382,13 @@ build() {
   [[ -d "${DEST_DIR}" ]] && rm -rf "${DEST_DIR}"
   mkdir -p "${DEST_DIR}" && log "Created build directory: ${DEST_DIR}"
 
+  # Need to declare this slightly more globally otherwise can't share it across functions 🤷
+  declare -A ALL_TAGS
```

Comment:
unironically caused me to scratch my head longer than I cared to admit

dunno why, but other variables just seem to work out of the box, but something with the `declare` syntax makes that not the case here